tests/codebuild/run_all_sample_namespace_tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+source create_tests.sh
+source delete_tests.sh
+source inject_tests.sh
+source feature_tests.sh
+
+crd_namespace="${1}"
+
+run_canary_tests "${crd_namespace}"
+verify_canary_tests "${crd_namespace}"
+run_feature_namespaced_tests "default" # Deploy job to namespace without operator.
```

Comment:
Just in case we, for whatever reason, mess up the uninstallation of the cluster scoped operator, we should name this something that we know won't work. Perhaps replace `"default"` with another random namespace like `"empty-namespace"`.

Reply:
done